rpm/util-linux.spec

```diff
@@ -32,23 +33,23 @@ Patch0:         0001-libblkid-probe-Disable-CD-ROM-probing-in-blkid_probe.patch
 
 ### Obsoletes & Conflicts & Provides
 # docs no longer generated due to needing lots of dependencies via rubygem-asciidoctor
-Obsoletes: util-linux-doc <= 2.38.1
-Provides: util-linux-doc = %{version}-%{release}
+Obsoletes:      util-linux-doc <= %{version}
+Provides:       util-linux-doc = %{version}-%{release}
```

Comment:
Can just drop these at this point. If there's been a stop release since introduction of package replacements, we can rely that the packages have been removed already.

Also wouldn't think there should be reasons to adjust the versioning here.

Reply:
The docs workaround was merged in 8af88071560c0e56ee52264c2a73105d211f0e49 in October 2022, and it was in 4.6.0,  so it can be dropped.

The versioning change was to automate the version bump, but I'll be dropping it anyway.